### ISSUE TITLE
Accessibility: Button missing aria-expanded

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#51898]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

The screen reader does not announce the state of the <button> element that opens and closes the form to change the shipping address. Screen reader users also do not know if triggering the button has successfully opened or closed the form.

![Image](https://github.com/user-attachments/assets/768d09bb-5cac-43d6-9b7e-837da9ec9454)

### Component or Page: 

Shortcode: [woocommerce_cart] & Cart Classic Block

### Name affected element:

Change address button

### Relevant code snippets:

`<a href="#" class="shipping-calculator-button">Change address</a>`

### Recommended fix:

Use the aria-expanded attribute to indicate the current state of the button. Set its initial value to ""false"" and toggle it between ""false"" and ""true"" with JavaScript. When implemented correctly, the screen reader will announce changes between expanded and collapsed states.
 
 For more information see https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded
